notebooks/ShotDataInferredStats.jl.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
Just checked out the branch and tried to run NB, 

Couple notes on the whole (it's a pain to locate code locs in the .ipynb format)

1. There's like a couple typos (input_file_path vs file_path in the df initialization cell)
2. Run NB start to finish to get it in a state to commit
3. Not that it's at all a big deal rn but float64 is totally overkill, we can use float32 with no real downside
4. I don't think you're actually using any BLAS functionality here